INN/umbrella-mstoday
# Remove background color from SCAIP

## Changes

This pull request makes the following changes:

- Removes the background color from SCAIP widget areas in posts

Before:
![screencapture-mstoday-test-2020-02-23-state-could-be-paying-10-percent-of-cost-instead-of-100-percent-for-some-community-mental-health-patients-2020-05-22-09_43_20](https://user-images.githubusercontent.com/18353636/82674021-b1864d00-9c10-11ea-936b-7cd4d74d5661.png)

After:
![screencapture-mstoday-test-2020-02-23-state-could-be-paying-10-percent-of-cost-instead-of-100-percent-for-some-community-mental-health-patients-2020-05-22-09_43_02](https://user-images.githubusercontent.com/18353636/82674056-bfd46900-9c10-11ea-9838-77b72c5843f6.png)

## Why



For #76

## Testing/Questions

Features that this PR affects:

- SCAIP widget areas in articles


Questions that need to be answered before merging:


Steps to test this PR:

1. View an article where SCAIP widgets are inserted and verify they have no background color
